### DISCUSSION TITLE
blog/2024-01-06-All-Features/

### DISCUSSION THREAD

**Discussion by giscus** [Akashic101/blog#2] [Announcements]
# blog/2024-01-06-All-Features/

This article is about: All the neat little features this blog has to offer

https://blog.davidmoll.net/blog/2024-01-06-All-Features/

**Comment by Akashic101** [comment]
Hey look, you found the comments. Pretty neat isn't it?

**Comment by bobmonsour** [comment]
Good stuff, David...but you have your [] and () backwards for the link you included for the broken link checker.

**Reply by Akashic101** [reply]
Good find, I must have missed that one.
